src/deployment/android.md

```diff
@@ -551,6 +551,7 @@ The resulting app bundle or APK files are located in
 ### Are there any special considerations with add-to-app?
 {% endcomment %}
 
+[Android plugin for Gradle]: {{site.android-dev}}/studio/releases/gradle-plugin
```

Comment:
Did you intentionally want this to link to the release notes? If not, https://developer.android.com/build seems like a better target for the text "Android plugin for Gradle".